package.json

```diff
@@ -4,7 +4,7 @@
     "email": "info@sussol.net"
   },
   "name": "mSupplyMobile",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
We will release as 2.0.0, if the version number gets whack all we have to do is commit back to 2.0.0 and publish.

That said, I'd prefer something like 2.0.0-rc1 until release we increment the rc number.